src/locale/locales/pt-BR/messages.po

```diff
@@ -1134,7 +1134,7 @@ msgstr "Você gostaria de dizer alguma coisa?"
 
 #: src/view/screens/Settings.tsx:498
 msgid "Dim"
-msgstr ""
+msgstr "Mais escuro"
```

Comment:
Dim não é mais escuro e sim menos escuro do que Dark.

![image](https://github.com/bluesky-social/social-app/assets/11423362/349d15a4-2d67-464d-a6b8-c16b94fbaf1c)

https://wikidiff.com/dim/dark

Reply:
Fixed! Twitter uses "Um pouco escuro", so I've settled for "Menos escuro" 

![image](https://github.com/bluesky-social/social-app/assets/1339236/20b8d30a-a311-45f6-ad37-06abda02c6bb)